validator_client/validator_services/src/block_service.rs

```diff
@@ -446,29 +445,75 @@ impl<S: ValidatorStore + 'static, T: SlotClock + 'static> BlockService<S, T> {
         //
         // Try the proposer nodes last, since it's likely that they don't have a
         // great view of attestations on the network.
-        let unsigned_block = proposer_fallback
```

Comment:
This comment could probably be updated since it's a bit outdated now that we try SSZ on all responsive nodes first, and then do a second run with JSON